tests/jvm/src/test/scala/scala/meta/tests/semanticdb/TargetedSuite.scala

```diff
@@ -971,20 +972,16 @@ object Compat {
     s"Stream.ConsWrapper#`#::`($name)."
   }
 
-  val ListBufferDeclsInString = "+43 decls"
+  val ListBufferDeclsInString = "+39 decls"
   lazy val ListBufferDeclsActual: String = {
-    val consWrapper = typeOf[scala.collection.mutable.ListBuffer[_]]
-    val decls = consWrapper.decls.size
-    s"+$decls decls"
+    if (scalaVersion.startsWith("2.11")) "+36 decls"
```

Comment:
note to self, we should move 2.11/2.12 specific tests to scala-2.11/scala-2.12 specific source directories